src/Admin/DevToolsUserAccess.php

```diff
@@ -56,39 +110,70 @@ public function register_rest_field() {
 		);
 	}
 
+	/**
+	 * Add the developer tools checkbox to the user edit screen.
+	 *
+	 * @param WP_User $profile_user Current user being edited.
+	 */
+	public function print_personal_options( $profile_user ) {
+		if ( ! current_user_can( 'edit_user', $profile_user->ID ) || ! AMP_Validation_Manager::has_cap( $profile_user ) ) {
+			return;
+		}
+		?>
+		<tr>
+			<th scope="row"><?php esc_html_e( 'AMP Developer Tools', 'amp' ); ?></th>
+			<td>
+				<label for="amp_dev_tools_enabled">
+					<input name="<?php echo esc_attr( self::USER_FIELD_DEVELOPER_TOOLS_ENABLED ); ?>" type="checkbox" id="amp_dev_tools_enabled" value="true" <?php checked( $this->is_user_enabled( $profile_user ) ); ?> />
```

Comment:
Why the `value="true"` attribute?

Reply:
Because enabling the checkbox means that developer tools being enabled should be `true`. In reality, it could be omitted and then `on` will be supplied. Both will have the same effect when the `$_POST` data is processed here:

https://github.com/ampproject/amp-wp/blob/c38f5372c8a5c8eb59a616d544c056fca5cb4ea5/src/Admin/DevToolsUserAccess.php#L147